ci/tests/puppeteer/scenarios/pm-must-change-password/script.json

```diff
@@ -0,0 +1,44 @@
+{
+  "dependencies": "core,ldap,pm-ldap,pm-webflow",
```

Comment:
Just an FYI: core is unnecessary to include, unless it's the only module and perhaps not even then. In earlier versions of this, there was a requirement to include at least one module in the scenario descriptor. Should no longer be the case.

Reply:
I had some noclassdeffound errors when I started working on it and tried adding that to fix but it turned out to be me needing to run gradle clean.